src/test/java/net/openhft/chronicle/wire/LongConversionExampleC.java

```diff
@@ -0,0 +1,52 @@
+package net.openhft.chronicle.wire;
+
+import net.openhft.chronicle.bytes.Bytes;
+import net.openhft.chronicle.bytes.FieldGroup;
+import java.nio.ByteBuffer;
+
+import static net.openhft.chronicle.bytes.Bytes.*;
+import static net.openhft.chronicle.wire.WireType.*;
+
+import net.openhft.chronicle.core.pool.ClassAliasPool;
+
+public class LongConversionExampleC {
+
+    static {
+        ClassAliasPool.CLASS_ALIASES.addAlias(House.class);
+    }
+
+    public static class House extends SelfDescribingMarshallable {
+        @FieldGroup("address")
+        private long text4a, text4b, text4c, text4d, text4e;
```

Comment:
I suggestion we  - add a comment that  this is, 5 longs each at 8 bytes = 40 bytes, so we can in this example store String with up to 40 ISO-8859 characters